File: FizzBuzz/fizzbuzz.py
#! /usr/bin/python3
#FizzBuzz in Python using Python3

for i in range(100):
    if i % 3 == 0 and i % 5 == 0:
        print("FizzBuzz")
    elif i % 3 == 0:
        print("Fizz")
    elif i % 5 == 0:
        print("Buzz")
    else:
        print("%d" % (i))

File: FizzBuzz/fizzbuzz.cpp
/*A pretty common way of coding fizzbuzz using for loops and else if statements*/
#include<iostream>

int main(void) {
    for(int i = 1; i < 101; i++) {
        if(i % 3 == 0 && i % 5 == 0) {
            std::cout<<"FizzBuzz\n";
        }
        else if(i % 3 == 0) {
            std::cout<<"Fizz\n";
        }
        else if(i % 5 == 0) {
            std::cout<<"Buzz\n";
        }
        else {
            std::cout<<i<<'\n';
        }
    }

    return 0;
}